### PR TITLE
Rpm/azure mgmt compute 0.18.7

### PR DESCRIPTION
**Pull request by apuntamb**
Supported Versions:

 * Nightly
 * 1.23
 * 1.22
 * 1.21

### CODE REVIEW

**Comment by apuntamb** [comment]
@evgeni @ekohl I have created this PR in order to separate the dependencies due to TimeOut in #4032 . Here, I can see the following error logs:
```
package: tfm-rubygem-azure_mgmt_compute-0.18.7-1.el7.noarch from downloaded_rpms
[2019-10-22T07:51:13.491Z]       unresolved deps:
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) < 0:0.12
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) >= 0:0.11.1
[2019-10-22T07:51:13.491Z]     package: tfm-rubygem-azure_mgmt_network-0.19.0-1.el7.noarch from downloaded_rpms
[2019-10-22T07:51:13.491Z]       unresolved deps:
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) < 0:0.12
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) >= 0:0.11.1
[2019-10-22T07:51:13.491Z]     package: tfm-rubygem-azure_mgmt_resources-0.17.6-1.el7.noarch from downloaded_rpms
[2019-10-22T07:51:13.491Z]       unresolved deps:
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) < 0:0.12
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) >= 0:0.11.1
[2019-10-22T07:51:13.491Z]     package: tfm-rubygem-azure_mgmt_storage-0.17.10-1.el7.noarch from downloaded_rpms
[2019-10-22T07:51:13.491Z]       unresolved deps:
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) < 0:0.12
[2019-10-22T07:51:13.491Z]          tfm-rubygem(ms_rest_azure) >= 0:0.11.1
```
But, `ms_rest_azure-0.11.1` is already merged in #4179 now. Thoughts?

**Comment by evgeni** [comment]
@apuntamb that is fine, we didn't publish the new rpms yet, I'll do that now and the errors should go away.

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -330,7 +330,6 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
-      <packagereq type="default">nodejs-jquery-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml]: this seems odd?

**Comment by evgeni** [comment]
The only weird thing I see it the removal of the `nodejs-jquery-doc` package from the comps. Probably needs to be before the RGD-START comment so that the auto-generator doesn't remove it again?

ACK otherwise.

**Comment by apuntamb** [comment]
> The only weird thing I see it the removal of the `nodejs-jquery-doc` package from the comps. Probably needs to be before the RGD-START comment so that the auto-generator doesn't remove it again?
> 
> ACK otherwise.

Should I manually add it prior to the `RGD-START` comment and amend it to the commit or some other way?

**Comment by evgeni** [comment]
> Should I manually add it prior to the `RGD-START` comment and amend it to the commit or some other way?

Yeah, that sounds good.

**Comment by evgeni** [comment]
@apuntamb I've published the new gems from the other PR now, so the next `repoclosure` run should be green.

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -326,11 +326,11 @@
       <packagereq type="default">rubygem-rb-inotify</packagereq>
       <packagereq type="default">rubygem-rsec</packagereq>
       <packagereq type="default">rubygem-rubyipmi</packagereq>
+      <packagereq type="default">nodejs-jquery-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml]: @apuntamb I am sorry, your original diff was actually correct. the `nodejs-jquery-doc` package was removed in 37cd622924b34a5033baee81791829f9927329b4, so you can safely remove this

Comment (apuntamb) [reply comps/comps-foreman-rhel7.xml]: done. :)

**Comment by apuntamb** [comment]
> @apuntamb I've published the new gems from the other PR now, so the next `repoclosure` run should be green.

Yes, now green! :)

**Comment by evgeni** [comment]
```
$ koji add-pkg foreman-plugins-nightly-rhel7 tfm-rubygem-azure_mgmt_compute tfm-rubygem-azure_mgmt_network tfm-rubygem-azure_mgmt_resources tfm-rubygem-azure_mgmt_storage --owner=kojiadm
Adding 4 packages to tag foreman-plugins-nightly-rhel7
```

**Pull request merged**